policyengine_us/variables/gov/states/ar/tax/income/exemptions/ar_retirement_or_disability_benefits_exemptions_eligible_person.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_us.model_api import *
+
+
+class ar_retirement_or_disability_benefits_exemptions_eligible_person(
+    Variable
+):
+    value_type = float
+    entity = Person
+    label = "Arkansas individual retirement or disability benefits exemption eligibility"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=13"
+    defined_for = StateCode.AR
+
+    def formula(person, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.ar.tax.income.exemptions.retirement_or_disability_benefits
+
+        head_eligibility = (
```

Comment:
make a structure:
age = person("age", period)
head = ..
spouse = ...
head_or_spouse = head | spouse 
return head_or_spouse * age

Reply:
I changed it to the structure like this, may I ask to make sure that this is what you mean?        
age = (person("age", period) >= p.age_threshold).astype(int)
head = person("is_tax_unit_head", period).astype(int)
spouse = person("is_tax_unit_spouse", period).astype(int)
return (head | spouse) * age

Thank you!

Reply:
yes, perfect